src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,70 @@
+package mate.academy;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import mate.academy.exceptions.DataProcessingException;
+
+public class ConnectionUtil {
```

Comment:
Util classes better to replace to util package

Reply:
++, thanks, moved it to the package